### PR TITLE
Feature/all entries page

### LINKED ISSUE #2: All Entries Page
## Value proposition

**As an** user
**I want** to see all my entries that i made
**So that** I can view them

## Description

Image

## Acceptance criteria

- [x] without entries show a placeholder "no entries yet"
- [x] I want to see a `list` of my entries, if there are any
- [x] I want to be able to `click` on the entry to show its content on the entry-page
- [x] I want to see a `back button` on the entry-page, that takes me back to all entries

## Tasks

- [x] Add a `heading`
- [x] Add a `list` with entries
- [x] `if` the list is empty, show a placeholder
- [x] have the list item to show its name and date of the entry

### PR DESCRIPTION
**Pull request by vaupunkt**
Added All Entries Page from [US2](https://github.com/vaupunkt/cbaspCompanion/issues/2)

### CODE REVIEW

**Comment by SelcoD** [comment]
LGTM !

**Review by dimitriosxmi — APPROVED** [review]
## LGTM!
Notes: Nice and clean code good job! 😄

File: pages/entries/[id].js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const [id, setId] = useState("");
+  useEffect(() => {
+    setId(router.query.id);
+    setEntry(
+      placeholderEntries.find((placeholderEntry) => placeholderEntry.id === id)
+    );
+  });
+
+  return <>{entry && <Entry data={entry} />}</>;
+}
```
Comment (djfarly) [inline pages/entries/[id].js:19]: I think this is too complicated.

You can get rid of both states and the effect and simply use a `const`.

```jsx
export default function EntryPage() {
  const router = useRouter();

  const entry = placeholderEntries.find((placeholderEntry) => placeholderEntry.id === router.query.id);

  return <>{entry && <Entry data={entry} />}</>;
}
```

Comment (vaupunkt) [reply pages/entries/[id].js:19]: somehow i am pretty sure i tried that but it didn't work, thats why i used useState and useEffect. 
But you are of course 100% right and it works 😅
thanks for looking over my code! 🙏🏻

**Pull request merged**